### PR TITLE
mixed_mich_2017: new_study_added

### PR DESCRIPTION
**Pull request by yichaoS**
# What?
Fix # https://github.com/cBioPortal/datahub/issues/537 (new study)
Original Publication: https://www.ncbi.nlm.nih.gov/pubmed/28783718

Cancer studies updated in this pull request:
- mixed_mich_2017

Triage Testing Instance: 
https://triage.cbioportal.mskcc.org/study/summary?id=metastatic_solid_tumors_mich_2017

### CODE REVIEW

**Comment by jjgao** [comment]
I think for `mixed` studies, maybe we don't have to use it in the study id. For this study, maybe we can use `metastatic_solid_tumors_mich_2019` or something like that? @yichaoS @ritikakundra

**Comment by jjgao** [comment]
@ritikakundra I remember that was some plan to change `mixed` to something else (`pancancer`?) in oncotree. Are we still going to do that?

File: public/mixed_mich_2017/meta_study.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+type_of_cancer: mixed
+cancer_study_identifier: mixed_mich_2017
+name: Mixed Metastasis 500 (Michigan U 2017)
```
Comment (rmadupuri) [inline public/mixed_mich_2017/meta_study.txt]: Should we rename the study from `Mixed Metastasis 500 (Michigan U 2017)` to `Metastatic Solid Cancers (UMich, Nature 2017)`

File: public/mixed_mich_2017/meta_study.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+short_name: MET500 (Michigan U 2017)
+description: Whole-exome and -transcriptome sequencing of 500 adult patients with metastatic solid tumor/primary normal pairs of diverse lineage and biopsy site.
```
Comment (rmadupuri) [inline public/mixed_mich_2017/meta_study.txt]: `-` infront of `transcriptome`

Comment (yichaoS) [reply public/mixed_mich_2017/meta_study.txt]: it is quoted form the publication directly. I think the `-` in front just means another `whole`

File: public/mixed_mich_2017/meta_study.txt (hunk 1 above)
Comment (yichaoS) [reply public/mixed_mich_2017/meta_study.txt]: @rmadupuri updated.

**Comment by yichaoS** [comment]
@rmadupuri "I think around 10 oncotree codes are old/not present in the current tree. update them to the ones we are using?" How did you find the old ones? Could you let me know?

**Comment by rmadupuri** [comment]
@yichaoS here are the 10 oncotree codes from the clinical file that are old/not present/not uppercase. 
SARC
MISC
MISC
Lung
TMT
SARC
Carcinoma
Lung
OV
CDRDD

**Comment by yichaoS** [comment]
@ritikakundra @rmadupuri Just emailed author about the missing CNA and RNA-Seq data issue. Also asked in the curation channel about converting the locations so circle CI can pass.

**Pull request merged**